### PR TITLE
Rum 18632 rum 18800 circle ci publish

### PR DESCRIPTION
**Pull request by robertdumitrescu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ SDK for generating and publishing events that capture user, application and tech
```
Comment (khaleqi67) [inline README.md:13]: @robertdumitrescu link and package name in NPM section needs to be updated

File: README.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ SDK for generating and publishing events that capture user, application and tech
 
 <p align="center"><img src="https://static1.squarespace.com/static/5a123416bce176a964daebe5/t/5aa18123c83025fedf718a51/1554300899717/?format=1500w"></p>
```
Comment (khaleqi67) [inline README.md:14]: @robertdumitrescu Browser compatibility needs to be updated

**Review by khaleqi67 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (robertdumitrescu) [reply README.md:14]: Will do it under another merge request as per our verbal conversation.

File: README.md (hunk 1 above)
Comment (robertdumitrescu) [reply README.md:13]: Will do it under another merge request as per our verbal conversation.

**Pull request merged**